src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/AbstractButtonBaseTests.cs

```diff
@@ -65,8 +59,10 @@ public virtual void ButtonBase_OverChangeRectangle_Get(Type controlType, Appeara
         Rectangle overChangeRectangle;
 
         // ButtonBase.Adapter prohibits this
-        if (appearance == Appearance.Normal && (flatStyle != FlatStyle.Standard && flatStyle != FlatStyle.Popup &&
-                                                flatStyle != FlatStyle.Flat))
+        if (appearance == Appearance.Normal
+            && (flatStyle != FlatStyle.Standard
+            && flatStyle != FlatStyle.Popup
+            && flatStyle != FlatStyle.Flat))
```

Comment:
```suggestion
        if (appearance == Appearance.Normal
            && (flatStyle != FlatStyle.Standard
                && flatStyle != FlatStyle.Popup
                && flatStyle != FlatStyle.Flat))
```

Indentation should reflect the level of the statement.